### PR TITLE
i18n(fr): Add `components/file-tree.mdx` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description

Add `components/file-tree.mdx` in French language

### CODE REVIEW

File: docs/src/content/docs/fr/components/file-tree.mdx [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    - Title.astro
+  - pages/
+
+</FileTree>
+
+</Preview>
+
+### Mettre en évidence les entrées
+
+Faire ressortir un fichier ou un répertoire en mettant son nom en gras, par exemple `**README.md**`.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/components/file-tree.mdx:92]:
```suggestion
Faites ressortir un fichier ou un répertoire en mettant son nom en gras, par exemple `**README.md**`.
```

File: docs/src/content/docs/fr/components/file-tree.mdx [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+- src
+  - components
+    - Header.astro un fichier **important**
+    - Title.astro
+
+</FileTree>
+
+</Preview>
+
+### Ajouter des espaces vierges
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/components/file-tree.mdx:177]: I feel like the previous version was a bit more explicit compared to mixing "espace vierge" and "remplacement".

```suggestion
### Ajouter des entrées fictives
```

File: docs/src/content/docs/fr/components/file-tree.mdx [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+Ajoutez des fichiers et des répertoires de remplacement en utilisant soit `...` soit `…` comme nom.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/components/file-tree.mdx:179]:
```suggestion
Ajoutez des fichiers et des répertoires fictifs en utilisant soit `...` soit `…` comme nom.
```

File: docs/src/content/docs/fr/components/file-tree.mdx [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+
+</FileTree>
+
+</Preview>
+
+## Props de `<FileTree>`
+
+**Implémentation :** [`FileTree.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/user-components/FileTree.astro)
+
+Le composant `<FileTree>` n'accepte aucun props.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/components/file-tree.mdx:225]:
```suggestion
Le composant `<FileTree>` n'accepte aucune props.
```

To match the discussion from yesterday on Discord.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation 🌟 

Left a few tiny suggestions but except for those, this looks great to me.

**Comment by thomasbnt** [comment]
> Thanks for the translation 🌟
> 
> Left a few tiny suggestions but except for those, this looks great to me.

Thanks @HiDeoo for the review!

**Review by HiDeoo — APPROVED** [review]
Amazing, thanks for the quick updates :rocket:

**Pull request merged**